src/locales/en_CA/location/postcode_by_state.ts

```diff
@@ -0,0 +1,21 @@
+// https://en.wikipedia.org/wiki/Postal_codes_in_Canada
+// The basic format is A1A 1A1
+// First letter is province/territory-specific
+// Other letters do not allow D, F, I, O, Q, U
+const suffix = '[0-9][A-HJ-NPRSTVW-Z] [0-9][A-HJ-NPRSTVW-Z][0-9]';
```

Comment:
The letters D and F are not allowed.

```suggestion
const suffix = '[0-9][ABCEGHJ-NPRSTVW-Z] [0-9][ABCEGHJ-NPRSTVW-Z][0-9]';
```